src/ore/Cargo.toml

```diff
@@ -87,7 +87,7 @@ tokio = { version = "1.32.0", features = ["macros", "rt-multi-thread"] }
 tokio-test = "0.4.2"
 
 [features]
-default = ["tokio-console", "workspace-hack"]
+default = ["tokio-console", "workspace-hack", "tracing-subscriber"]
```

Comment:
Is this a heavy default feature? Why does it need to be added to the default set?

Reply:
its required for `cargo test` to work in the mz-ore crate, but i suppose i can use `--features=tracing-subscriber` and leave it off for now